bigquery_etl/glean_usage/templates/cross_channel_events_unnested.view.sql

```diff
@@ -0,0 +1,16 @@
+-- Generated via ./bqetl glean_usage generate
+CREATE OR REPLACE VIEW
+    `{{ project_id }}.{{ target_view }}`
+AS
+{% for (dataset, channel) in datasets -%}
+{% if not loop.first -%}
+UNION ALL
```

Comment:
`UNION ALL` on Fenix `event` views doesn't work since the ordering of the columns is different.

Reply:
I left a comment in https://bugzilla.mozilla.org/show_bug.cgi?id=1710088 to discuss our options